src/spectral/rulesets/.defaultsForSpectral.yaml

```diff
@@ -92,3 +92,15 @@ rules:
   oas3-schema: off
   # Turn off - duplicates non-configurable validation in base validator
   oas3-unused-components-schema: off
+
+  # custom Spectral rule to ensure content object contains schema
+  content-entry-contains-schema:
+    description: Content entries in request and response bodies must specify a schema
+    given:
+    - $.paths[*].[post,put,patch].requestBody.content[*]
+    - $.paths[*].[get,post,put,patch,delete].responses[*].content[*]
+    - $.components.responses[*].content[*]
```

Comment:
I'm not that well versed in Spectral syntax but didn't Mike make a comment earlier about how `components` didn't need to be checked? Do you know if this line is necessary?

Reply:
Yes, thank you for catching!